src/Shared/Reporting/ColorFormatter.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Extensions.Tools.Internal
+{
+    public class ColorFormatter : IFormatter
```

Comment:
Yeah it would be good to consolidate these things in a single place. Can you open an issue to track this?